### ISSUE TITLE
Bug: Scheduling post obscure error

### ISSUE THREAD

**Issue by cobbspur** [TryGhost/Ghost#7099]
### Issue Summary
When attempting to schedule a post less than 2 minutes into the future, an error notification is provided with no details as to what went wrong:

![](http://puu.sh/q2rKq/fefca50858.png)

Error is shown in console:

![](http://puu.sh/q2rLE/1945cabb49.png)

### Steps to Reproduce

1. Load a draft post in Ghost-Admin
2. Open PSM and select time 1 minute in future.
3. Save Draft 
4. Schedule Post

### Technical details:

* Ghost Version: https://github.com/TryGhost/Ghost/commit/71282a241e76608df5d99112df005ebbc7a3204b
* Node Version: 4.4.7

**Comment by AileenCGN** [comment]
I'm not able to reproduce this.

When I select a draft post and set the publish time to be 1 minute in future, click on `Save Draft`, then my publish time get reseted to what it was before. I don't have the `Schedule Post` option.

I was able to reproduce the following situation:
1. Load draft post
2. Open PSM and select 3 minutes in future
3. Save Draft
4. Wait 2 minutes
5. Schedule Post

Usually, this should (and did, when we merged it in first place) output the servers' error message in the red notification like this:

![image](https://cloud.githubusercontent.com/assets/8037602/16875804/9df76284-4aa1-11e6-862f-d6c4f1040a15.png)

I'm on it now and try to find out where it goes wrong...
